insights/parsers/du.py

```diff
@@ -0,0 +1,59 @@
+"""
+du - Command
+============
+
+Module for the processing of output from the ``du`` command stored in
+file.  The base class ``DiskUsage`` provides all of the functionality.
+
+This example demonstrates the funcionality of ``DiskUsage`` class.
+
+Examples:
+    >>> FILE_CONTENT = '''
+    ... COMMAND> du -sh /var/lib/mongodb
+    ...
+    ... 22G     /var/lib/mongodb
+    ... '''
+    >>> du_info = shared[DiskUsage]
+    >>> du_info.get_size('/var/lib/mongodb')
+    22G
+    >>> du_info.get_size('/var/lib/pgsql')
+    None
+
+"""
+
+from .. import Parser, parser, LegacyItemAccess
+from .. import get_active_lines
+
+
+@parser("disk_space")
+class DiskUsage(LegacyItemAccess, Parser):
+    """
+    Typical content of `mongodisk_space` command looks like:
```

Comment:
There should be a double colon `::` at the end of this line to properly format the following lines.